openy_calc/src/Form/CalcBlockForm.php

```diff
@@ -127,42 +127,56 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#markup' => $header,
     ];
 
+    $form['membership_calc_body'] = [
+      '#type' => 'container',
+      '#attributes' => [
+        'class' => ['membership-calc__body'],
+      ],
+    ];
+
     switch ($step) {
       case 1:
         // Membership type step.
-        $form['type'] = [
+        $form['membership_calc_body']['type'] = [
           '#element_variables' => $types,
           '#subtype' => 'membership_type_radio',
           '#type' => 'calc_radios',
-          '#title' => $this->t('Which option best describes the type of membership you need?'),
+          '#title' =>  '<h3>' . $this->t('Choose your membership type') . '</h3>',
```

Comment:
Better to use Markup::
`        '#title' => Markup::create('<h3>' . $this->t('Choose your membership type') . '</h3>'),
`

Reply:
Fixed